src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/contactperson/ContactPersonDpsApiMockServer.kt

```diff
@@ -150,4 +174,26 @@ class ContactPersonDpsApiMockServer : WireMockServer(WIREMOCK_PORT) {
         ),
     )
   }
+  fun stubGetContactPhone(contactPhoneId: Long, response: SyncContactPhone = contactPhone()) {
+    stubFor(
+      get("/sync/contact-phone/$contactPhoneId")
+        .willReturn(
+          aResponse()
+            .withStatus(201)
```

Comment:
typo on the status here? GET=200

Reply:
Yes, I'll correct in next PR
Actually I think they return 200 for POSTs as well